packages/react-router-dom/__tests__/DataBrowserRouter-test.tsx

```diff
@@ -12,25 +12,27 @@ import {
   prettyDOM,
 } from "@testing-library/react";
 import "@testing-library/jest-dom";
+import {
+  useNavigate,
+  UNSAFE_DataRouterStateContext,
+  UNSAFE_resetModuleScope,
+} from "react-router";
 
 import {
   DataBrowserRouter,
   DataHashRouter,
+  Form,
+  Link,
   Route,
   Outlet,
   useLoaderData,
   useActionData,
   useRouteError,
   useNavigation,
-  Form,
-  Link,
   useSubmit,
   useFetcher,
   useFetchers,
 } from "../index";
-import { _resetModuleScope } from "react-router/lib/components";
```

Comment:
Had to import this from the same package name import above or else we weren't resetting the correct module scope. this would go away if we went to relative imports and we could avoid the need for the public API.